### DISCUSSION TITLE
Will MAUI export WASM like Uno Platform ? (That is a very important feature)

### DISCUSSION THREAD

**Discussion by cmorait** [dotnet/maui#8929] [General]
I am frustrated on choosing a path for a multi-platform app.

MAUI was the way to go (for Andoid, IOs, Windows and Mac) but lucks a very important feature to be able to export on WASM. That makes you the need to write an entire web app with other technologies. 

UNO Platform has that option. I hope Microsoft to consider that feature too and make linux support official.  

Does someone know if WASM is in the plans at a future upgrade of MAUI ?

Thanks.

**Comment by xtuzy** [comment]
Current no plan, according to https://twitter.com/davidortinau/status/1528549030531301376?t=5UnZRFETkh_xlP5utwY24Q&s=19
Maybe you can see community if have progress, see https://twitter.com/praeclarum/status/1544002881468084224?t=hw1cTgncNyarK-uGJznihw&s=19

**Reply by manishbisen** [reply]
Interesting question
